service/history/historyEngine.go

```diff
@@ -2986,7 +2986,7 @@ func (e *historyEngineImpl) ReapplyEvents(
 		currentExecution,
 		func(wfContext execution.Context, mutableState execution.MutableState) (*updateWorkflowAction, error) {
 			// Filter out reapply event from the same cluster
-			toReapplyEvents := make([]*workflow.HistoryEvent, len(reapplyEvents))
+			var toReapplyEvents []*workflow.HistoryEvent
```

Comment:
add a `0` in the middle, for less mem allocation
```go
toReapplyEvents := make([]*workflow.HistoryEvent, 0, len(reapplyEvents))
```

Reply:
updated